RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java

```diff
@@ -976,6 +1000,23 @@ private void getFormAddressArtifacts(AbstractFile webDataFile, String dbFilePath
         }
     }
 
+    /**
+     * Check the type of the object and if it is bytes then it is encrypted and return the string and
+     * set flag that field and file are encrypted
+     * @param dataValue Object to be checked, the object is from a database result set
+     * @return the actual string or an empty string
+     */
+    private String isFieldEncrypted(Object dataValue) {
```

Comment:
The name makes me think this method will return a boolean. Maybe rename to processField()? Or something else?

Reply:
Changed